README.md

```diff
@@ -1,89 +1,98 @@
 # Atlas gene annotations from Ensembl
 
-This repository contains scripts that extract gene attributes from large JSON dumps provided by Ensembl for a range of species. Gene attributes extracted from JSON populate the corresponding fields in an output TSV file (a gene-by-annotation table).
+This repository contains scripts that extract gene attributes from Ensembl JSON dumps for a range of species. Gene attributes extracted from JSON populate the corresponding fields of an output TSV file (a gene-by-annotation table).
 
 The final aggregated gene annotation file is loaded into Solr indexes as a bioentities collection. To run the scripts, please make sure the `/bin` directory is in the PATH. The following dependencies need to be installed: 
 
 - awk
 - jq (1.5)
-- bats (for testing)
+- bats (testing)
+- nextflow (workflow execution)
 
-### Testing the scripts
-run `atlas_annotations_ensembl_run_tests.sh` in order to execute the tests. 
-
-### Extract annotations from JSON dumps to TSV formatted files 
-
-#### Entry points
-`test/run_annotations_from_ensembl.sh`
-The entry point to trigger the conversion of JSON Ensembl annotations to TSV formatted annotations for all the species defined in the `./annsrcs/ensembl` This script runs bsub runs on LSF cluster for each species on parallel for efficiency.`TODO: include /annsrcs/wbps`
-
-By triggering the entry script `test/run_annotations_from_ensembl.sh` you will effectively run the script `bin/annotations_from_ensembl.sh` for each species configuration file defined in `annsrcs/ensembl`
-
-Before running the entry point script it is important to set environmental variables as prerequisite.
+### Required env variables
+The following environment variables need to be defined for the scripts to run correctly (see example data under paths provided): 
 ```
-## prerequisites - export environmental variables
-## Note : These paths are used for testing purpose. While in production, we need these output files to be dumped in traditional $ATLAS_PROD/bioentity_properties/annotations 
 export ENSEMBL_JSON_PATH=/hps/nobackup2/production/ensembl/ensprod/search_dumps/release-101b/vertebrates/json (pilot file path provided by Mark from Ensembl for testing)
 export ANNOTATIONS_PATH=/ebi/microarray/home/suhaib/json_Ensembl/annotations
+export GENE_ATTRIBUTES_PATH=/ebi/microarray/home/suhaib/json_Ensembl/ensembl
```

Comment:
Please do not use real internal paths for public files in repos, this is a security issue. You should have internal environment files that point to the correct places, but here.

Reply:
@pcm32 sure, I'll remove them.